CHANGELOG.rst

```diff
@@ -154,6 +166,16 @@ Features
 
 Fixes
 ~~~~~
+- 2422 Hide private keys in configuration endpoint response.
+- Hyperledger#2492: Fixes not all triggers being executed that match an
+  event.
+- 2504 Fix failing tps benchmark.
+- 2477 Fix bug when permissions from roles weren't counted.
+- 2416 Fix lints on macOS arm.
+- 2457 Fix tests flakiness related to shut down on panic. #2457: Add shut down on panic configuration
+- 2473 parse rustc --version instead of RUSTUP_TOOLCHAIN.
+- 1480 Shut down on panic. #1480: Add panic hook to exit program on panic
+- 2376 Simplified Kura, no async, two files.
```

Comment:
```suggestion
- 2422 Hide private keys in configuration endpoint response.
- 2492: Fix not all triggers being executed that match an
  event.
- 2504 Fix failing tps benchmark.
- 2477 Fix bug when permissions from roles weren't counted.
- 2416 Fix lints on macOS arm.
- 2457 Fix tests flakiness related to shutting down on panic.
- 2473 Parse rustc --version instead of RUSTUP_TOOLCHAIN.
- 1480 Add panic hook to exit programs on panic.
- 2376 Simplify Kura: no async, two files.
```